lib/rubocop/config_loader.rb

```diff
@@ -113,6 +113,11 @@ def configuration_from_file(config_file)
         config = load_file(config_file)
         return config if config_file == DEFAULT_FILE
 
+        if ConfigLoader.ignore_parent_exclusion
+          print 'Ignoring AllCops/Exclude from parent folders' if debug?
+          return merge_with_default(config, config_file)
```

Comment:
Done. However it comes with a few strings attached. The reason I did that way was to avoid violating (a) getting perceived complexity too high, if I add else clause; or (b) exceed the maximum class side if I move the else into another method.

For now I moved the else to another method (looks cleaner) and increased the maximum class size.